packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -792,13 +792,16 @@ const groupPage = (
         <Grid item xs={12} md={6}>
           <EntityGroupProfileCard variant="gridItem" />
         </Grid>
+        <Grid item xs={12} md={6}>
+          <EntityLinksCard />
+        </Grid>
```

Comment:
I know this is just the example but I would do the same here to keep things consistent, move it down 👍